pkg/frontend/frontend.go

```diff
@@ -189,6 +194,20 @@ func (f *frontend) authenticatedRoutes(r *mux.Router) {
 
 	s.Methods(http.MethodGet).HandlerFunc(f.getOpenShiftClusters).Name("getOpenShiftClusters")
 
+	s = r.
+		Path("/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/{resourceProviderNamespace}/{resourceType}/{resourceName}/toppods").
+		Queries("api-version", "{api-version}").
+		Subrouter()
+
+	s.Methods(http.MethodGet).HandlerFunc(f.topOpenShiftPods).Name("topOpenShiftPods")
+
+	s = r.
+		Path("/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/{resourceProviderNamespace}/{resourceType}/{resourceName}/topnodes").
+		Queries("api-version", "{api-version}").
+		Subrouter()
+
+	s.Methods(http.MethodGet).HandlerFunc(f.topOpenShiftNodes).Name("topOpenShiftNodes")
```

Comment:
All geneva actions should have `/admin` prefix for [`AdminClientAuthorizer`](https://github.com/Azure/ARO-RP/blob/4a48003b3e2345fda51ac3e860df4134cb494158/pkg/frontend/middleware/authenticated.go#L25) to kick in.

Exception: we have few geneva actions which reuse same endpoints available publicly to customers via ARM (put or patch) and do not have `/admin` prefix. But these endpoints are admin only so must have the prefix.

Reply:
I will make this change

Reply:
@m1kola top pods and top nodes geneva actions now have the `/admin` prefix. Unit tests have been updated accordingly.